skimage/restoration/deconvolution.py

```diff
@@ -16,16 +16,16 @@ def wiener(image, psf, balance, reg=None, is_real=True, clip=True):
 
     Parameters
     ----------
-    image : (M, N) ndarray
-       Input degraded image
+    image : ndarray
+       Input degraded image (can be N dimensional).
```

Comment:
Not sure if we have used it consistently, but I prefer the hyphenated form
```suggestion
       Input degraded image (can be n-dimensional).
```

Reply:
I must have taken it from the RL function docstring. Changing it to lower case here and there.

It'd great to have a document with a consistent scheme for reference in these cases.